src/containers/layouts/dashboards/trainings/trainings1/easy1.component.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import {
+  ListRenderItemInfo,
+  ScrollView,
+  View,
+  ViewProps,
+} from 'react-native';
+import {
+  withStyles,
+  ThemeType,
+  ThemedComponentProps,
+  StyleType,
+} from '@kitten/theme';
+import { Exercise } from '@src/core/model/exercise.model';
+import {
+  TrainingCard1,
+  TrainingCardProps,
+} from '@src/components/trainings/trainingCard.component';
+import {
+  ViewPager,
+  List,
+} from '@kitten/ui';
+import { Text } from '@src/components/common';
+
+interface ComponentProps {
+  exercises: Exercise[];
+  onTrainingDetails: (index: number) => void;
+}
+
+interface State {
+  selectedPagerIndex: number;
+}
+
+export type Easy1ComponentProps = ThemedComponentProps & ComponentProps;
+
+class Easy1Component extends React.Component<Easy1ComponentProps, State> {
+
+  public state: State = {
+    selectedPagerIndex: 0,
+  };
+
+  private onChangePager = (index: number): void => {
+    this.setState({ selectedPagerIndex: index });
```

Comment:
can be shorthanded if `index` arg will be called `selectedPagerIndex`

Reply:
You definitely like "sugar" :) refactored